Vagrantfile

```diff
@@ -0,0 +1,70 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"). You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2016 Adam Stevko. All rights reserved.
+#
+
+Vagrant.configure("2") do |config|
+  # Every Vagrant development environment requires a box. You can search for
+  # boxes at https://atlas.hashicorp.com/search.
+  config.vm.box = "openindiana/hipster"
+
+  # Disable automatic box update checking. If you disable this, then
+  # boxes will only be checked for updates when the user runs
+  # `vagrant box outdated`. This is not recommended.
+  config.vm.box_check_update = true
+
+  # Unless OpenIndiana is better supported in vagrant, we have to use this
+  # workaround. The problem is with vagrant creating folder as root:root,
+  # but rsync connects as vagrant and fails to write.
+  config.vm.synced_folder ".", "/vagrant", type: "rsync",
+    rsync__rsync_path: "pfexec rsync", owner: "vagrant", group: "vagrant"
```

Comment:
Does this still execute rsync with --delete ? if so any changes in the vm will be deleted everytime the rsync starts.